syntax/parser.go

```diff
@@ -2274,7 +2274,7 @@ loop:
 			}
 			fallthrough
 		default:
-			p.curErr("a command can only contain words and redirects")
+			p.curErr("a command can only contain words and redirects. This command stopped at token: '%s'", p.tok)
```

Comment:
error messages are a single sentence. how about:

    "a command can only contain words and redirects; encountered %s", p.tok)

Reply:
Sure, updated